shallow_backup/reinstall.py

```diff
@@ -73,6 +73,13 @@ def reinstall_configs_sb(configs_path):
 		elif os.path.isfile(path_to_backup):
 			copyfile(path_to_backup, dest_path)
 
+	# reinstall crontab
+	with open(os.path.join(configs_path, "crontab.txt"), "r") as f:
```

Comment:
Why can't we just reinstall it with `$ crontab crontab.txt`?

Reply:
`$ crontab crontab.txt` just replaces the current crontab with the contents of the file. The used command only ads lines which aren't yet present thus not removing anything.

Reply:
Ok, I'm cool with that.